client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -145,6 +146,42 @@ function CheckoutSummaryFeaturesListDomainItem( { domain } ) {
 	);
 }
 
+function CheckoutSummaryPlanFeatures() {
+	const translate = useTranslate();
+	const hasDomainsInCart = useHasDomainsInCart();
+	const planInCart = usePlanInCart();
+	const planFeatures = getPlanFeatures( planInCart, translate, hasDomainsInCart );
+
+	return (
+		<>
+			{ planFeatures &&
+				planFeatures.map( ( feature ) => {
+					return (
+						<CheckoutSummaryFeaturesListItem>
```

Comment:
This probably needs a `key` prop. Since `feature` is (I think) a string, you should be able to use it.